Makefile

```
@@ -1,81 +1,14 @@
 NAME    := fuse
+DEB_NAME := fuse3
 SRC_EXT := gz
 SOURCE   = https://github.com/libfuse/libfuse/archive/$(NAME)-$(VERSION).tar.$(SRC_EXT)
-PATCHES := $(NAME).conf $(NAME)-install-nonroot.patch $(NAME)-linux-ioctl.patch
-
-COMMON_RPM_ARGS := --define "%_topdir $$PWD/_topdir"
-DIST    := $(shell rpm $(COMMON_RPM_ARGS) --eval %{?dist})
-ifeq ($(DIST),)
-SED_EXPR := 1p
+ID_LIKE := $(shell . /etc/os-release; echo $$ID_LIKE)
+COMMON_PATCHES := $(NAME)-install-nonroot.patch $(NAME)-linux-ioctl.patch
```

I was thinking the more straightforward:

```suggestion
PATCHES := $(NAME)-install-nonroot.patch $(NAME)-linux-ioctl.patch
```